HomeTask2/basicOp.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * evaluates the expression formed with binary operator and numbers
+ * @param {string} operation Any character from , '+','-','*','/'
+ * @param {number} value1 number 1
+ * @param {number} value2 number 2
+ * @example basicOp('+', 4, 7)
+ * 
+ * @returns {number} evaluation of the expression
+ */
+
+function basicOp(operation, value1, value2) {
+    const operators = {
+        '+': (number1, number2) => {
+            return number1 + number2;
```

Comment:
If you just do a simple transformation or calculation, you need not put a return statement. Instead, you can do it as (num1, num2) => num1 + num2. Can avoid usage of unnecessary brackets and return statements.

Reply:
Done changes as per the request